.mergify.yml

```diff
@@ -62,6 +62,10 @@ pull_request_rules:
           - check-success=e2e (3.11, 1.75, rust_vendor)
           - check-success=e2e (3.12, 1.75, rust_vendor)
 
+          - check-success=e2e (3.10, 1.75, download_sequence)
```

Comment:
It's not a big deal, but I've been trying to keep these in alphabetical order based on the test name.

Reply:
Understood Ill update this if I make another file touching this file.